src/component/common/EnvironmentSplash/EnvironmentSplashPage/EnvironmentSplashPage.tsx

```diff
@@ -0,0 +1,24 @@
+interface EnvironmentSplashPageProps {
+    title: React.ReactNode;
+    topDescription: React.ReactNode;
+    image?: React.ReactNode;
+    bottomDescription?: React.ReactNode;
+}
+
+const EnvironmentSplashPage = ({
```

Comment:
If all this component is doing is providing the containing div, perhaps we don't actually need a component for this at all? It becomes a pretty meaningless abstraction at that point.